### PR TITLE
add 172. Factorial Trailing Zeroes

### PR DESCRIPTION
**Pull request by nayuta-ai**

### CODE REVIEW

File: 172_Factorial_Trailing_Zeroes.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# time O(1)
+# space O(1)
+class Solution:
+    def trailingZeroes(self, n: int) -> int:
+        cnt = 0
+        for i in range(1,6):
+            if n < 5**i:
+                break;
+            cnt = cnt + n//(5**i)
```
Comment (tkkawa) [inline 172_Factorial_Trailing_Zeroes.py:9]: cnt += n // (5 ** i)

File: 172_Factorial_Trailing_Zeroes.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# time O(1)
+# space O(1)
+class Solution:
+    def trailingZeroes(self, n: int) -> int:
+        cnt = 0
+        for i in range(1,6):
+            if n < 5**i:
+                break;
```
Comment (tkkawa) [inline 172_Factorial_Trailing_Zeroes.py:8]: insert a blank line after break statement

**Review by tkkawa — CHANGES_REQUESTED** [review]

File: 172_Factorial_Trailing_Zeroes.py (hunk 1 above)
Comment (nayuta-ai) [reply 172_Factorial_Trailing_Zeroes.py:9]: - insert space

File: 172_Factorial_Trailing_Zeroes.py (hunk 2 above)
Comment (nayuta-ai) [reply 172_Factorial_Trailing_Zeroes.py:8]: - insert a blank

File: 172_Factorial_Trailing_Zeroes.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# time O(1)
+# space O(1)
+class Solution:
+    def trailingZeroes(self, n: int) -> int:
+        cnt = 0
+        for i in range(1,6):
+            if n < 5 ** i:
+                break;
+                
+            cnt = cnt + n // (5 ** i)
```
Comment (tkkawa) [inline 172_Factorial_Trailing_Zeroes.py:10]: cnt += n // (5 ** i)

**Review by tkkawa — CHANGES_REQUESTED** [review]

**Review by tkkawa — APPROVED** [review]
LGTM

**Pull request merged**